src/it/projects/MJAVADOC-591/src/main/java/com/example/Foo.java

```diff
@@ -0,0 +1,31 @@
+package com.example.foo;
```

Comment:
change package to `com.example`, otherwise the integration-testwill fail.

Reply:
Done, thanks.